webapp/src/Constants.ts

```diff
@@ -50,4 +50,12 @@ export const Constants = {
         MANIFEST_PATH: '/.well-known/ai-plugin.json',
     },
     KEYSTROKE_DEBOUNCE_TIME_MS: 250,
+    MemoriesStoreTypes: {
```

Comment:
nit: let's align with the naming in webapi here?
```suggestion
    MemoryStoreType: {
````

Reply:
The webapi uses plural as well.

Reply:
ah I was looking at `GetMemoryType()` in `OptionsController` before, but I see it's changed. imo we should align on `memoryStoryType`, but I don't really have a strong opinion on that. I think alignment around something would be good tho. right now we use `MemoriesStoreTypes` here, `getMemoriesStoreType` in `useChat`, and `memoryStoreType` in the new route.

Reply:
ahhh ok, I see `MemoriesStoreType` defined on the backend now... I think I missed it before. ignore 😄